pkg/daemon/daemon.go

```diff
@@ -319,8 +319,8 @@ func (dn *Daemon) ClusterConnect(
 			if usingEviction {
 				verbStr = "Evicted"
 			}
-			glog.Info(fmt.Sprintf("%s pod from Node", verbStr),
-				"pod", fmt.Sprintf("%s/%s", pod.Name, pod.Namespace))
+			glog.Info(fmt.Sprintf("%s pod from Node ", verbStr),
```

Comment:
Not a blocker but there is `glog.Infof()` to do that in one go.  Also the "from Node" part seems redundant, of course we're evicting a pod from a node.

How about just:
```
glog.Infof("%s pod %s/%s", verbStr, pod.Name, pod.Namespace);
```

Reply:
lol great minds colin!!!

Reply:
ooh i missed taking out the sprintf on my next to last pass now done.